templates/drupal9/ce-dev/ansible/deploy.yml.j2

```diff
@@ -42,6 +42,7 @@
         # Create a list of URLs to test with LHCI
         test_urls:
           - "https://{{ _domain_name }}"
+    - cachetool_bin: "/usr/local/bin/cachetool"
```

Comment:
This could be `/home/ce-dev/.bin/cachetool` I think? But maybe not with ce-provision defaults - worth checking?

Reply:
I tried with it actually, but for some reason the .bin/cachetool.phar is always the latest version of opcache. I don't know who/what installs this phar , maybe it is part of Debian Bullseye?